src/Umbraco.Cms.Integrations.Testsite.V14/Umbraco.Cms.Integrations.Testsite.V14.csproj

```diff
@@ -12,11 +20,7 @@
   <ItemGroup>
     <!-- Opt-in to app-local ICU to ensure consistent globalization APIs across different platforms -->
     <PackageReference Include="Microsoft.ICU.ICU4C.Runtime" Version="72.1.0.3" />
-    <ProjectReference Include="..\Umbraco.Cms.Integrations.Crm.ActiveCampaign\Umbraco.Cms.Integrations.Crm.ActiveCampaign.csproj" />
-    <ProjectReference Include="..\Umbraco.Cms.Integrations.Crm.Dynamics\Umbraco.Cms.Integrations.Crm.Dynamics.csproj" />
-    <ProjectReference Include="..\Umbraco.Cms.Integrations.Automation.Zapier\Umbraco.Cms.Integrations.Automation.Zapier.csproj" />
-    <ProjectReference Include="..\Umbraco.Cms.Integrations.Crm.Hubspot\Umbraco.Cms.Integrations.Crm.Hubspot.csproj" />
-    <ProjectReference Include="..\Umbraco.Cms.Integrations.Search.Algolia\Umbraco.Cms.Integrations.Search.Algolia.csproj" />
+    <ProjectReference Include="..\Umbraco.Cms.Integrations.SEO.Semrush\Umbraco.Cms.Integrations.SEO.Semrush.csproj" />
```

Comment:
- I was wondering if we could remove these references in the test site, and later if someone want to test a package or anything in this repo, they can add the reference by themselves.

Reply:
Maybe leave them in but commented out?  That way it's easy to add the one you need for testing.

Reply:
My thought is that you can leave them all referenced, as with the completion of Semrush and merge into main, when someone would want to play with the integrations they would have the web environment ready to go.